installation/synology.md

```diff
@@ -3,6 +3,15 @@ layout: documentation
 title: Synology DiskStation
 ---
 
+:::tip Note
+
+openHAB 4 and higher need Java 17.
+You will need to provide a working Java installation on your DiskStation.
+The sections below are written for Java 11 but the docs are not updated currently, since we have now secured knowledge about how to install Java 17 on the DiskStation.
```

Comment:
```suggestion
The sections below are written for Java 11 but the docs are not updated currently, since we have not secured knowledge about how to install Java 17 on the DiskStation.
```